### ISSUE TITLE
Accessibility: Fix incorrect role / New messages button (from Emulator repo)

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#2698]
This is a duplicate of #2669. Closing this issue.

**Actual Result:**
The chat boxes are defined in the list but they are not accessible by up/down arrow key whereas they are accessible by tab only, due to tab access on each chat boxes the user will not be able to reach "New Message" button. 

**Expected Result:**
The chat boxes should be accessible by up and down arrow and on pressing tab from chat box the focus should jump on "New Message" button.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Open a Bot Dialog box gets opened.
5. Navigate to the "bot URL" edit box and enter URL- http://localhost:3979/api/messages
6. Enter "Microsoft App ID and Microsoft App Password". As well as check debug mode.
7. Navigate to "Connect" button and activate it.
8. "Debug" separate window appears.
9. Copy the "/INSPECT attach <GUID>” and paste it in M.S. Teams opened from "Azure Portal"
10. Chat with the created bot, the chat's done on "MS Teams" are appeared on the "Debug" screen.
11. Navigate to the chat boxes and observe the issue.

**User Impact:**
Keyboard user will have to use "Tab" key again and again, if the chat list have more data. 
As well as to reach to the next item i.e. "JSON", the user must be go through all the chat.  
More over the AT user will not be able to activate or use the functionality of "New Message" button. 

**Note:**
Same issue observed on "Live chat" screen.

**Observation:**
"New Message" button can be accessed by mouse. Moreover if it can be accessed by any predefined "Shortcut keys" it will be beneficial for the user.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Incorrect role is defined for the chat boxes.zip](https://github.com/microsoft/BotFramework-Emulator/files/3824298/Incorrect.role.is.defined.for.the.chat.boxes.zip)~
